ts/components/ModuleAttachment.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useCallback } from "react";
+import {
+  ActivityIndicator,
+  GestureResponderEvent,
+  Pressable,
+  PressableProps,
+  StyleSheet,
+  View
+} from "react-native";
+import {
+  IOColors,
+  IOIconSizeScale,
+  IOIcons,
+  IOListItemVisualParams,
+  IOScaleValues,
+  IOSpacingScale,
+  IOSpringValues,
+  IOStyles,
+  Icon,
+  LabelSmall,
+  VSpacer,
+  WithTestID
+} from "@pagopa/io-app-design-system";
+import Animated, {
+  Extrapolate,
+  interpolate,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSpring
+} from "react-native-reanimated";
+import Placeholder from "rn-placeholder";
+import I18n from "../i18n";
+
+type PartialProps = WithTestID<{
+  title: string;
+  iconName: IOIcons;
```

Comment:
Everything is perfect, but there's one last edit to make the substitution with the new component easier. It would be better to set the format with an `enum` and add a map object to associate the format with the relative icon name:
```suggestion
  format: "pdf" | "zip" | "etc…";
```
And the relative mapping (just a proposal, do whatever you want):
```js
const formatMap = {
  pdf: docAttachPDF,
  zip: etc…
}
```
What do you think?

Reply:
Great Idea! Changed https://github.com/pagopa/io-app/pull/5101/commits/f336df32fa7ad356f73602a5d26aa7603f74a248
Let me know if it's ok to use `doc` for generic format.